pandas/plotting/api.py

```diff
@@ -0,0 +1,20 @@
+"""
+Plotting api
```

Comment:
what I mean is, unless you explicity reference these imports, like ``from pandas.plotting import .....)``  you would by-default actually import anything.

thoughI guess for back-compat this is needed.

IOW, the imports for things like ``matplotlib`` would *only* be done when the pandas.plotting namespace is used (IOW, you try to plot something), so the import is done lazily later, rather than on startup now (to register converters and such).